tests/test_locality_NeighborList.py

```diff
@@ -228,6 +228,28 @@ def test_copy(self):
         npt.assert_equal(nlist.segments, nlist2.segments)
         npt.assert_equal(nlist.neighbor_counts, nlist2.neighbor_counts)
 
+    def test_ordering(self):
+        # default behavior sorts by (i, j, distance)
+        tuples = list(zip(self.nlist.query_point_indices,
+                          self.nlist.point_indices,
+                          self.nlist.distances))
+        sorted_tuples = list(sorted(tuples))
+
+        self.assertEqual(tuples, sorted_tuples)
+
+        # test sorting by (i, distance, j)
```

Comment:
Could you re-use the data from setUp if you made this a separate test?

Reply:
I don't immediately see a way to parameterize `setUp` in the unittest framework, did you have something in mind, or were you thinking to just always store two neighbor lists? Adding a parameter to it and just re-calling it would work, of course; let me know if you want it done that way.

Reply:
Don’t parameterize, just let setUp do its thing and then use self.nq to create a sorted nlist. You just don’t have to create a new box, points, nq because those already exist.

Reply:
Oh, of course! That was thoughtless of me.